geniza/footnotes/models.py

```diff
@@ -141,16 +140,49 @@ def __str__(self):
             else:
                 author = author_lastnames[0]
 
-        parts = []
-
         if self.title:
             # if this is an article, wrap title in quotes
             if self.source_type.type == "Article":
                 parts.append('"%s"' % self.title)
             else:
                 parts.append(self.title)
 
-        # TODO: formatted version with italics for book/journal title
+        if self.languages:
+            if not self.languages.all():
+                languages = None
+
+            if len(self.languages.all()) > 1:
+                # Separate languages with commas
+                languages = ",".join([l.name for l in self.languages.all()])
+            else:
+                languages = " ".join([l.name for l in self.languages.all()])
```

Comment:
the if/else not needed, you can just join the list — even if there's 1 or zero entries you get the right result... unless I'm missing something; why join with a space in the second case? I'm not sure that's doing anything.

```suggestion
            # Separate languages with commas
            languages = ", ".join([l.name for l in self.languages.all()])
```